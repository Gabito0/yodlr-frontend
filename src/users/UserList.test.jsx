import React from "react";
import { render } from "@testing-library/react";
import Users from "./UserList";
it("matches snapshot", function () {
  const { asFragment } = render(<Users />);
  expect(asFragment()).toMatchSnapshot();
});
