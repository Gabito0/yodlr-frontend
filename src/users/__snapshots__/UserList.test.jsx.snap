// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="d-flex justify-content-center align-items-center"
    style="height: 100vh;"
  >
    <div
      class="spinner-border text-primary"
      role="status"
    >
      <span
        class="visually-hidden"
      >
        Loading...
      </span>
    </div>
  </div>
</DocumentFragment>
`;
